### PR TITLE
vendor: consul-template v0.25.1

### PR DESCRIPTION
**Pull request by greut**
`consult-template` v0.25.1 adopted allow/denylist nomenclature, https://github.com/hashicorp/consul-template/pull/1388 - it's probably a good time to follow.

### CODE REVIEW

File: client/fingerprint_manager.go [hunk 1]
```diff
@@ -65,28 +65,28 @@ func (fm *FingerprintManager) getNode() *structs.Node {
...
 // each.
 func (fp *FingerprintManager) Run() error {
 	// First, set up all fingerprints
 	cfg := fp.getConfig()
-	whitelistFingerprints := cfg.ReadStringListToMap("fingerprint.whitelist")
-	whitelistFingerprintsEnabled := len(whitelistFingerprints) > 0
-	blacklistFingerprints := cfg.ReadStringListToMap("fingerprint.blacklist")
+	allowlistFingerprints := cfg.ReadStringListToMap("fingerprint.allowlist")
+	allowlistFingerprintsEnabled := len(allowlistFingerprints) > 0
+	denylistFingerprints := cfg.ReadStringListToMap("fingerprint.denylist")
```
Comment (notnoop) [inline client/fingerprint_manager.go:76]: Given that these are user visible configuration, it'd be nice to keep accepting old names (potentially with a warning) for backward compatibility but only document the new keys.

**Comment by tgross** [comment]
Closing in lieu of https://github.com/hashicorp/nomad/pull/8988

**Comment by greut** [comment]
@tgross @notnoop :man_shrugging: !?

**Comment by tgross** [comment]
> @tgross @notnoop 🤷‍♂️ !?

Yeah, sorry about that... I was reviewing someone else's work on updating consul-template to support Consul Namespaces, and didn't realize that we had an in-flight PR open. A bit of a miscommunication over here among the Nomad reviewers.

**Pull request closed without merging**